examples/src/main/java/com/opengamma/strata/examples/finance/CalibrationCheckExample.java

```diff
@@ -73,22 +74,22 @@
    * The location of the curve calibration groups file.
    */
   private static final ResourceLocator GROUPS_RESOURCE =
-      ResourceLocator.of(ResourceLocator.FILE_URL_PREFIX + PATH_CONFIG + "curves/groups.csv");
+      ResourceLocator.ofFile(new File(PATH_CONFIG + "curves/groups.csv"));
```

Comment:
Isn't it a bit retro to be using `File`? Why not

    ResourceLocator.ofPath(Paths.get(PATH_CONFIG).resolve("curves/groups.csv"));